packages/web-frontend/src/screens/LandingPage/LandingPage.js

```diff
@@ -53,7 +54,12 @@ export const LandingPage = () => {
       <TopBar />
       <Wrapper backgroundImage={backgroundImage}>
         <Container maxWidth={false}>
-          {projects.length > 1 ? <MultiProjectLandingPage /> : <SingleProjectLandingPage />}
+          {/* tupaia requires projects to work so we can assume that if there are no projects, it's just */}
+          {/* because they haven't loaded yet. We can replace this with more idiomatic loading state */}
+          {/* when we refactor to use react-query */}
+          {projects.length === 0 && <LoadingScreen isLoading background={null} />}
```

Comment:
I'm not sure if there is a better way to do this 3 way logic but I didn't want to use a nested ternary so I did this